app/mailers/admin_mailer.rb

```diff
@@ -1,15 +1,6 @@
 class AdminMailer < ApplicationMailer
   include HtmlCleaner
```

Comment:
I think this was only used for `strip_html_breaks_simple`, so you should be able to remove this.